custom/icds_reports/sqldata/exports/dashboard_usage.py

```diff
@@ -247,7 +249,7 @@ def get_excel_data(self):
         return [
             [
                 self.title,
-                excel_rows
+                sorted(excel_rows, key=lambda x:(x[1], x[2], x[3]))
```

Comment:
When you will sort it and the headers are also in the excel_rows, There is a chance that header line will be ended up not at the top. Can you fix this?
@challabeehyv